docs/uml-class-diagrams/database/production/DatabasePersistentStorageImplementation/README.md

```diff
@@ -0,0 +1,36 @@
+# UML Class Diagrams: edu.oakland.production.database.DatabasePersistentStorageImplementation
+
+**Primary Owner:** Cam'Ron Grant, Database Team SCRUM Master ([@camrongrant1](https://github.com/camrongrant1/))
+
+**Secondary Owners:**
+
+- Kyle Poterek, Database Team SCRUM Assistant Master ([@kylepoterek](https://github.com/kylepoterek/))
+- Mathew Yaldo, Database Team SCRUM Integrator ([@MathewYaldo](https://github.com/MathewYaldo/))
+
+## Purpose
+
+This class shall...
```

Comment:
Make sure you add a purpose for the class. It will be similar to that of the interace.